packages/xrpl/test/integration/index.ts

```diff
@@ -23,24 +26,30 @@ export * from './requests/accountTx.test'
 export * from './requests/bookOffers.test'
 export * from './requests/channelVerify.test'
 export * from './requests/depositAuthorized.test'
+export * from './requests/fee.test'
 export * from './requests/gatewayBalances.test'
 export * from './requests/ledger.test'
 export * from './requests/ledgerClosed.test'
 export * from './requests/ledgerCurrent.test'
 export * from './requests/ledgerData.test'
 export * from './requests/ledgerEntry.test'
-export * from './requests/submitMultisigned.test'
 export * from './requests/noRippleCheck.test'
 export * from './requests/pathFind.test'
 export * from './requests/ripplePathFind.test'
+export * from './requests/serverInfo.test'
 export * from './requests/submit.test'
```

Comment:
We're missing `./serverState.test` from this list (Unless it was intentionally excluded for some reason)

Reply:
Nice catch

Reply:
This is still missing from the file it seems.